docs/setup/upgrade/index.md

```diff
@@ -12,139 +12,38 @@ menu_name: docs_{{ .version }}
 section_menu_id: setup
 ---
 
-# Upgrading Stash
+# Upgrading KubeStash
 
-This guide will show you how to upgrade various Stash components. Here, we are going to show how to upgrade from an old Stash version to the new version, and how to update the license, etc.
+This guide will show you how to upgrade various KubeStash components. Here, we are going to show how to upgrade from an old KubeStash version to the new version, and how to update the license, etc.
 
-## Upgrading Stash from `v2021.xx.xx` to `{{< param "info.version" >}}`
+## Upgrading KubeStash from `v2021.xx.xx` to `{{< param "info.version" >}}`
```

Comment:
We don't need this headline.

Reply:
I've used `Upgrading KubeStash` title instead of `## Upgrading KubeStash from `v2021.xx.xx` to `{{< param "info.version" >}}``